app/utils/answersHelpers/consignment/houseConsignment/ConsignmentItemAnswersHelper.scala

```diff
@@ -57,4 +60,20 @@ class ConsignmentItemAnswersHelper(
     id = Some(s"change-net-weight-${houseConsignmentIndex.display}"),
     call = Some(Call(GET, "#"))
   )
+
+  def additionalReferences: Seq[SummaryListRow] =
+    getAnswersAndBuildSectionRows(AdditionalReferencesSection(houseConsignmentIndex, itemIndex)) {
+      additionalReferenceIndex =>
+        additionalReferenceRow(houseConsignmentIndex, itemIndex, additionalReferenceIndex)
+    }
+
+  def additionalReferenceRow(hcIndex: Index, itemIndex: Index, additionalReferenceIndex: Index): Option[SummaryListRow] =
```

Comment:
`hcIndex` and `itemIndex` should be available from the class signature I believe?